From 65b3ccc54ae53b9460cf4370fc0cad1063c4fee5 Mon Sep 17 00:00:00 2001
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Wed, 19 Nov 2014 11:29:44 +0000
Subject: [PATCH 21/29] Fix a missing \r in crash_generation_app

Fix a missing \r in crash_generation_app

I thought this made a difference, but it doesn't seem to...

Signed-off-by: Jon Turney <jon.turney@dronecode.org.uk>
---
 src/client/windows/tests/crash_generation_app/crash_generation_app.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/client/windows/tests/crash_generation_app/crash_generation_app.cc b/src/client/windows/tests/crash_generation_app/crash_generation_app.cc
index 52736768..93457277 100644
--- a/src/client/windows/tests/crash_generation_app/crash_generation_app.cc
+++ b/src/client/windows/tests/crash_generation_app/crash_generation_app.cc
@@ -254,7 +254,7 @@ static void ShowClientCrashed(void* context,
   line[0] = _T('\0');
   result = swprintf_s(line,
                       kMaximumLineLength,
-                      L"%s\n",
+                      L"%s\r\n",
                       str_line.c_str());
   if (result == -1) {
     delete[] line;
-- 
2.15.0

